python/src/aiconfig/editor/client/src/shared/prompt_schemas/HuggingFaceText2SpeechRemoteInferencePromptSchema.ts

```diff
@@ -13,6 +13,7 @@ export const HuggingFaceText2SpeechRemoteInferencePromptSchema: PromptSchema = {
         type: "string",
         description: `Hugging Face model to use. Can be a model ID hosted on the Hugging Face Hub or a URL 
         to a deployed Inference Endpoint`,
+        default: "suno/bark",
```

Comment:
Is this the one that fails by default? Maybe we should choose a different one and default it in the parser?

Reply:
<img width="909" alt="Screenshot 2024-01-26 at 5 36 12 PM" src="https://github.com/lastmile-ai/aiconfig/assets/141073967/cd0575aa-a20c-4ee0-a2fe-523e68abc7f2">

yep suno/bark requires a pro subscription. We could choose a different one but a user could just change the model